app/renderer/js/electron-bridge.ts

```diff
@@ -46,6 +47,28 @@ class ElectronBridge extends EventEmitter {
 	set_send_notification_reply_message_supported = (value: boolean): void => {
 		this.send_notification_reply_message_supported = value;
 	};
+
+	decrypt_clipboard = (): { key: string; check(): string | null } => {
+		const key = crypto.randomBytes(32);
+		return {
+			key: key.toString('hex'),
+			check() {
+				try {
+					const ciphertext = Buffer.from(clipboard.readText(), 'hex');
+					const decipher = crypto.createDecipheriv(
+						'aes-256-gcm',
+						key,
+						ciphertext.slice(0, 12),
+						{ authTagLength: 16 }
+					);
+					decipher.setAuthTag(ciphertext.slice(-16));
+					return decipher.update(ciphertext.slice(12, -16), undefined, 'utf8') + decipher.final('utf8');
```

Comment:
I would probably extract variables for the `ciphertext.slice()` sections all the start for readability, e.g. `const iv = ciphertext.slice(0, 12);`, etc.

Reply:
Done.